### ISSUE TITLE
Problem with shortcut `o.cpp 1`

### ISSUE THREAD

**Issue by Jofagi** [trovu/trovu#377]
I'm sent to an intermediate "Weiterleitungshinweis" page that warns about forwarding.

![grafik](https://github.com/trovu/trovu/assets/6442817/35ed33db-9cfd-4f07-b576-9d8d8146a237)

The trovu query to get the above result is `cpp sprintf`
<br/>
<hr/>

<details><summary>This repo is using Opire - what does it mean? 👇</summary><br/>💸Everyone can add rewards for this issue commenting <code>/reward 100</code> (replace <code>100</code> with the amount)<br/>💪 If someone starts working on this issue to earn the rewards, they can comment <code>/try</code> to let everyone know!<br/>🙌 And when they open the PR, they can comment <code>/claim #377</code> either in the PR description or in a PR's comment<br/><br/>👀 Also, everyone can tip any user commenting <code>/tip 20 @Jofagi</code> (replace <code>20</code> with the amount, and <code>@Jofagi</code> with the user to tip)<br/><br/>If you want to learn more, go to our <a href="https://docs.opire.dev">documentation</a></details>

**Comment by georgjaehnig** [comment]
Thanks for letting me know. I use the "Google Feeling Lucky" URL here, and it used to redirect directly to the first search result. Seems that changed.

Let me know if there's a better URL I could use.

The official search of the website points to DDG:

    https://duckduckgo.com/?sites=cppreference.com&q=sprintf&ia=web

but only shows search results. Let me know if that suits better (I'm not a C++ coder.)

**Comment by Jofagi** [comment]
Interesting... until now I also used that DDG search, which presents a list of which you have to pick an entry. 
I just learned that in DDG you can prepend a backslash to the search term to emulate the "feeling lucky" behavior: https://duckduckgo.com/duckduckgo-help-pages/results/syntax/

- So you could switch to DDG here and let the user decide whether to prepend a `\` to go to the first match. This is easy enough to type but assumes that the user knows about that DDG feature.
- Or you could always prepend the backslash in trovu but then there is no way of having a fuzzy search.

That probably depends on what philosophy trovu follows here...

Maybe use DDG without backslash but add a hint to the `cpp` command description?

**Comment by georgjaehnig** [comment]
Thanks for the hint about the "Feeling Lucky" of DDG. Since it works more reliably than the Google one, I now chose this one.

**Comment by Jofagi** [comment]
Sorry, doesn't work as advertised :) --  The `&ia=web` in the DDG query seems to suppress the `\` effect. Right now it shows the result list instead of going to the first result.

**Comment by georgjaehnig** [comment]
Thanks again for the info.

I debugged now for a while. Seems DDG checks the referrer, and does not redirect when it's set. 

So in https://trovu.net/process/index.html?#query=debug%3Acpp+sprintf, I've added 
```html
<meta name="referrer" content="no-referrer">
```
It's deployed already and should work as expected.

If not, click the link above and make sure the HTML source contains this line.

And let me know here if it's still not working.
